src/test/java/buisness/core/HomePage.java

```diff
@@ -0,0 +1,14 @@
+package buisness.core;
+
+import com.codeborne.selenide.Selenide;
+
+import static core.utils.services.ConfigurationService.getProperty;
+
+public class HomePage {
+
+  public void openHomePage() {
```

Comment:
Is it reasonable to have HomePage? I recommend you to perform an open page in @Before hook